schema_cache.rst

```diff
@@ -0,0 +1,131 @@
+.. _schema_cache:
+
+Schema Cache
+============
+
+Certain PostgREST features need information from the database schema; however, accessing the database directly would be too costly, that is why PostgREST generates a database schema cache on startup and gets the information from it instead:
```

Comment:
```suggestion
PostgREST caches metadata from the database schema to avoid repeating expensive queries. This metadata is not required by all of the PostgREST features, only the following:
```